snap/snapcraft.yaml

```diff
@@ -492,34 +492,6 @@ parts:
   # from edgex-go which at this point will fail
   # eventually we would like these changes to be upstreamed but until then
   # they live here. see also https://github.com/edgexfoundry/edgex-go/issues/1109
-  go112:
-    plugin: nil
-    source: snap/local/build-helpers
-    override-build: |
-      # use dpkg architecture to figure out our target arch
-      # note - we specifically don't use arch
-      case "$(dpkg --print-architecture)" in
-        amd64)
-          FILE_NAME=go1.12.7.linux-amd64.tar.gz
-          FILE_HASH=66d83bfb5a9ede000e33c6579a91a29e6b101829ad41fffb5c5bb6c900e109d9
-          ;;
-        arm64)
-          FILE_NAME=go1.12.7.linux-arm64.tar.gz
-          FILE_HASH=4da1f7198a8fa0c4067852656b6c10153a4eca5a26aca28ef02ae9f4a7939ba5
-          ;;
-      esac
-      # download the archive, failing on ssl cert problems
-      curl https://dl.google.com/go/$FILE_NAME -O
-      echo "$FILE_HASH $FILE_NAME" > sha256
-      sha256sum -c sha256 | grep OK
-      mkdir -p $SNAPCRAFT_STAGE/go1.12
-      tar -C $SNAPCRAFT_STAGE/go1.12 -xf "$FILE_NAME" --strip-components=1
-    prime:
-      - "-*"
-    # build after kong to prevent staging go binaries into the snap as they
-    # are large - see the TODO on the kong part
```

Comment:
can you move this comment down to the `after` section on go115 please?

Reply:
and restored